docs/spec.md

```diff
@@ -1884,46 +1884,30 @@ Produces an `output` tensor from a constant `value`.
 Performs an element-wise conversion from one element type to another on
 `operand` tensor and produces a `result` tensor.
 
-For conversions involving **integer-to-integer**, if there is an unsigned/signed
-overflow, the result is implementation-defined and one of the following:
-
-* mathematical result modulo $2^n$, where n is the bit width of the result,
-  for unsigned overflow. For signed integer overflow, wraps the result around
-  the representable range $[-2^{n-1},\ 2^{n-1} - 1]$.
-* saturation to $2^{n-1} - 1$ (or $-2^{n-1}$) for signed overflow and
-  saturation to $2^n - 1$ (or $0$) for unsigned overflow.
-
-For conversions involving **floating-point-to-floating-point** or
-**integer-to-floating-point**, if the source value can be exactly represented in
-the destination type, the result value is that exact representation. Otherwise,
-the behavior is TBD ([#180](https://github.com/openxla/stablehlo/issues/180)).
-
-Conversion involving **complex-to-complex** follows the same behavior of
-**floating-point-to-floating-point** conversions for converting real and
-imaginary parts.
-
-For conversions involving **floating-point-to-complex** or
-**complex-to-floating-point**, the destination imaginary value is zeroed or the
-source imaginary value is ignored, respectively. The conversion of the real part
-follows the **floating-point-to-floating-point** conversion.
+For **boolean-to-any-supported-type** conversions, the value `false` is
+converted to zero, and the value `true` is converted to one. For
+**any-supported-type-to-boolean** conversions, a zero value is converted to
+`false`, and non-zero values are converted to `true`. See below for how this
+work for complex types.
```

Comment:
How about moving booleans to the top, to follow the standard order: booleans > integers > floats > complex numbers?